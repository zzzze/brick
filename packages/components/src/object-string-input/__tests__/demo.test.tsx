import demoTest from '@brick/tools/tests/demoTest'

demoTest('components/src/object-string-input')
