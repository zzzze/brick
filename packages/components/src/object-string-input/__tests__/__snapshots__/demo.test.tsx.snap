// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./packages/components/src/object-string-input/demo/basic.md correctly 1`] = `<textarea />`;

exports[`renders ./packages/components/src/object-string-input/demo/with-props.md correctly 1`] = `
<textarea
  name="obj"
>
  {"foo":"bar","baz":123}
</textarea>
`;
